apps/enc.c

```diff
@@ -243,6 +246,11 @@ int enc_main(int argc, char **argv)
                 goto opthelp;
             cipher = c;
             break;
+        case OPT_ITER:
```

Comment:
The standard itself gives an example of 1000. Not sure what is considered best practice these days. @vdukhovni do you have a view?